.changeset/chilly-eagles-know.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/presentation-hierarchies": minor
+---
+
+`createHierarchyProvider`: Added ability to specify whether hierarchy should be expanded to filtering path target, when specifying the `filtering.paths` prop. With this change, hierarchy is no longer expanded to `filtering.paths` targets by default.
```

Comment:
Please add a code example, similar to my suggestion in changelog for `hierarchies-react`.

Reply:
Added.